src/Raven.Server/Documents/DatabaseRequestHandler.cs

```diff
@@ -157,5 +165,68 @@ protected void AddPagingPerformanceHint(PagingOperationType operation, string ac
             if(ShouldAddPagingPerformanceHint(numberOfResults))
                 Database.NotificationCenter.Paging.Add(operation, action, details, numberOfResults, pageSize, duration, totalDocumentsSizeInBytes);
         }
+
+        protected DynamicJsonValue GetCustomConfigurationAuditJson(string name, BlittableJsonReaderObject configuration)
+        {
+            switch (name)
```

Comment:
I wonder if we want to call this if `if (LoggingSource.AuditLog.IsInfoEnabled == false)` ? Maybe we can move this into `LogTaskToAudit()` ?

Reply:
Done. decided to make the check before calling the function (instead moving it in) to keep the convention like we do with logging to TrafficWatch

Reply:
Done